docs/pages/tutorial/create-your-first-app.mdx

```diff
@@ -41,13 +41,19 @@ It will create a new project directory and install all the necessary dependencie
 
 This command will create a new directory for the project with the name: **StickerSmash**.
 
-Let's also <A href="/static/images/tutorial/sticker-smash-assets.zip" download>download the assets</A> that we will need throughout this tutorial. After downloading the archive, unzip it and replace the existing **assets** directory with it in the project directory. This will override the default assets when the new project is initialized.
+</Step>
+
+<Step label="2">
+
+## Download assets
+
+We'll need some assets to build the app. Let's <A href="/static/images/tutorial/sticker-smash-assets.zip" download>download these assets</A> that we will need throughout this tutorial. After downloading the archive, unzip it and replace the existing **assets** directory with it in the project directory. This will override the default assets when the new project is initialized.
```

Comment:
That's a nice improvement! 👍 

We can even use `BoxLink` at the top of this step with a nice icon to emphasize even more the download link, but it's not a necessary change, feel free to also leave the PR as is.

Reply:
Great suggestion! I've updated it (check screenshot) to use BoxLink as well as content in the section. Thank you!